### PR TITLE
Updated docs with the new storage plugin for Dell EMC PowerFlex/ScaleIO (formerly VxFlexOS)

### PR DESCRIPTION
**Pull request by sureshanaparti**
Added new storage plugin (Dell EMC PowerFlex/ScaleIO) to the documentation.

### CODE REVIEW

**Review by rhtyd — APPROVED** [review]
LGTM, @sureshanaparti can you link the scaleio setup to this link - https://www.dell.com/support/home/en-in/product-support/product/scaleio/docs

I think we don't need to document how to setup scaleio in CloudStack docs, but we can put a note and use this link to the sentence (for example) to say refer the docs...?

File: source/adminguide/storage.rst [hunk 1]
```diff
@@ -105,6 +105,7 @@ Storage media \\ hypervisor                    VMware vSphere   Citrix XenServer
 **Storage over-provisioning**                  NFS and iSCSI    NFS                  NFS                         No
 **SMB/CIFS**                                   No               No                   No                          Yes
 **Ceph/RBD**                                   No               No                   Yes                         No
+**PowerFlex/ScaleIO**                          No               No                   Yes                         No
```
Comment (rhtyd) [inline source/adminguide/storage.rst:108]: ** somewhere we need to mention the support is for ScaleIO/PowerFlex v3.5

**Comment by rhtyd** [comment]
cc @shwstppr can you review this as well thnx

**Review by shwstppr — COMMENTED** [review]
LGTM on changes other than point already raised by @rhtyd to specify supported version somewhere
@sureshanaparti

**Comment by shwstppr** [comment]
requesting docbuild

File: source/adminguide/storage.rst (hunk 1 above)
Comment (sureshanaparti) [reply source/adminguide/storage.rst:108]: updated version in the doc.

**Comment by rhtyd** [comment]
@sureshanaparti can you fix the conflict and kick a build test

**Comment by sureshanaparti** [comment]
requesting docbuild

**Review by shwstppr — APPROVED** [review]

**Comment by rhtyd** [comment]
We need lgtm from @borisstoyanov and @andrijapanicsb

**Comment by rhtyd** [comment]
The dev/code PR was merged, merging this.

**Pull request merged**